py3status/modules/tor_rate.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+"""
+Display the current transfer rates of a tor instance
+
+Configuration parameters:
+    cache_timeout: An integer specifying the cache life-time of the modules
+        output in seconds (default 2)
+    control_address: The address on which the Tor daemon listens for control
+        connections (default "127.0.0.1")
+    control_password: The password to use for the Tor control connection
+        (default None)
+    control_port: The port on which the Tor daemon listens for control
+        connections (default 9051)
+    format: A string describing the output format for the module
+        (default "↑ {up} ↓ {down}")
+    rate_unit: The unit to use for the transfer rates
+        (default "B/s")
+    si_units: A boolean value selecting whether or not to use SI units
+        (default 0)
+    value_format: A string describing how to format the transfer rates
+        (default "[\?min_length=12 {rate:.1f} {unit}]")
```

Comment:
`format_value` would be a better name.  You should document the placeholders for this format too.